src/styles/variables.scss

```diff
@@ -0,0 +1,8 @@
+$main-accent-color: #060b35;
+$text-secondary-color: #616070;
+$color1: #00acdc;
```

Comment:
Consider using more descriptive variable names instead of `$color1` and `$color2`. This will make your code more readable and maintainable.

Reply:
> Consider using more descriptive variable names instead of `$color1` and `$color2`. This will make your code more readable and maintainable.

Good point